docs/2.guide/3.going-further/1.events.md

```
@@ -0,0 +1,75 @@
+
+## Creating Events and Listeners
+
+You can create your own custom events using the `hook` method:
+
+``ts
+const nuxtApp = useNuxtApp()
+
+nuxtApp.hook('app:user:registered', payload => {
+  console.log('A new user has registered!', payload)
+})
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified
null

(MD040, fenced-code-language)